server-common/src/test/java/org/apache/kafka/server/common/FeaturesTest.java

```diff
@@ -24,21 +24,34 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import static org.apache.kafka.server.common.Features.validateDefaultValueAndLatestProductionValue;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class FeaturesTest {
     @ParameterizedTest
-    @EnumSource(Features.class)
+    @EnumSource(value = Features.class, names = {
+        "UNIT_TEST_VERSION_0",
+        "UNIT_TEST_VERSION_1",
+        "UNIT_TEST_VERSION_2",
+        "UNIT_TEST_VERSION_3",
+        "UNIT_TEST_VERSION_4",
+        "UNIT_TEST_VERSION_5"}, mode = EnumSource.Mode.EXCLUDE)
```

Comment:
Hmm, is it correct for `EXCLUDE` to be empty?

Reply:
I don't quite follow. Doesn't this mean the test exclude the values provided by `names`?